docs/install_guides/configurations-for-ncs2.md

```diff
@@ -11,68 +11,29 @@
 
```

Comment:
Maybe we can change the heading to "## Linux & Raspbian OS"? because Raspbian OS is treated separately in the install guides, I think we should also mention it explicitly to make it easier to find. #14666

Reply:
Actually Raspbian included in Linux* OSes family.